staging/src/k8s.io/apiserver/pkg/audit/policy/reader_test.go

```diff
@@ -71,6 +71,13 @@ rules:
   - level: Metadata
 `
 
+var testCases = []struct {
+	caseName, caseContent string
+}{
+	{`policyWithNoRule`, "apiVersion: audit.k8s.io/v1beta1\nkind: Policy"},
```

Comment:
I think for readability it's better to avoid manually writing '\n'. Maybe do something like this instead?

```
{
  "policyWithNoRule",
  `
apiVersion: audit.k8s.io/v1beta1
kind: Policy
`,
}
```